scripts/Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+aiogram = "==2.15"
+aiohttp = "==3.7.4.post0"
+async-timeout = "==3.0.1"
+google-api-core = "==2.1.1"
+google-api-python-client = "==2.26.1"
+google-auth = "==2.3.0"
+google-auth-httplib2 = "==0.1.0"
+google-auth-oauthlib = "==0.4.6"
+googleapis-common-protos = "==1.53.0"
+httplib2 = "==0.20.1"
+oauth2client = "==4.1.3"
+oauthlib = "==3.1.1"
+pylint = "==2.7.4"
+black = "==20.8b1"
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```

Comment:
We use python 3.8 now. Are you sure that version 3.9 won't initiate some problems?